pkgs/servers/nosql/influxdb2/default.nix

```
@@ -1,4 +1,4 @@
-{ buildGoModule
+{ buildGo118Module
```

This is no longer needed, nixpkgs now uses go1.18 by default
```suggestion
{ buildGoModule
```